feat(card): Allow slotted html title (#471)
adobe/spectrum-web-components: packages/card/README.md

</sp-card>
```

## Title

> The default application of a title for an `sp-card` is to leverage the `title` attribute. When HTML based title content is desired, the `title` slot is also available for deliverin this content.

Reviewer: ```suggestion
By default, the title for an `sp-card` is applied via the `title` attribute, which is restricted to string content only. When HTML content is desired, a slot named `title` available for applying the title.
```

English is the most difficult language we work in...

Author: lol, sounds good to me. done and done!
